Features.md

```diff
@@ -21,7 +21,7 @@ This is a (non-exhaustive) list of features currently (un)supported by CHIP-SPV.
   hipDeviceDisablePeerAccess, hipDeviceGetStreamPriorityRange,
   hipStreamGetPriority, hipDevicePrimaryCtxRelease, hipDevicePrimaryCtxRetain,
   hipDevicePrimaryCtxSetFlags, hipMemPrefetchAsync, hipMemAdvise,
-  hipModuleLoadData, hipModuleUnload, hipModuleLaunchKernel
+  hipModuleLoadData, hipModuleUnload, hipModuleLaunchKernel, hipStreamPerThread
```

Comment:
I merged the hipStreamPerThread and resolved the conversation. This is supported now

Reply:
ah ok, i'll remove the commit.